http/oidc/src/main/java/org/wildfly/security/http/oidc/ElytronMessages.java

```diff
@@ -278,5 +278,11 @@ interface ElytronMessages extends BasicLogger {
 
     @Message(id = 23070, value = "Authentication request format must be one of the following: oauth2, request, request_uri.")
     RuntimeException invalidAuthenticationRequestFormat();
+
+    @Message(id = 23071, value = "Invalid %s")
+    String invalidSessionRandomValue(String name);
```

Comment:
Maybe we could rename this to invalidNonceValue(String nonce) and update the message itself to "Invalid nonce %s".

Reply:
done